### DISCUSSION TITLE
Can I download WSL2 instead of WSL?

### DISCUSSION THREAD

**Discussion by Fardeen26** [Cyfrin/foundry-full-course-cu#1993] [Q&A] [answered]
In the setup videos Patrick downloaded the WSL, but as the most updated can I download WSL2?
Is there any differences in WSL and WSL version 2?

**Comment by EngrPips** [comment]
Honestly, I haven't used either of the two to know if there are differences, but I assume `WSL2` should work fine

**Reply by Fardeen26** [reply]
Thank you for your input! I'll give WSL2 a try and see how it works.

**Comment by Podziuu** [comment, marked as answer]
Yes, you can definitely download WSL2 and yes, there are some differences between WSL and WSL2. You can read about them [here](https://learn.microsoft.com/en-us/windows/wsl/compare-versions).

**Reply by EngrPips** [reply]
Thanks for sharing this with us.

**Reply by Fardeen26** [reply]
Thank you for the clarification and the resource! I'll go ahead with WSL2 :)
